ezyfox-server-core/src/main/java/com/tvd12/ezyfoxserver/api/EzyAbstractResponseApi.java

```diff
@@ -32,13 +37,40 @@ protected final void normalResponse(
             return;
         }
         Object bytes = encodeData(pack.getData());
+        EzyConstant transportType = pack.getTransportType();
         if (immediate) {
             for (EzySession session : recipients) {
-                session.sendNow(createPacket(bytes, pack));
+                EzyPacket packet = null;
+                try {
+                    packet = createPacket(
+                        session,
+                        transportType,
+                        bytes
+                    );
+                    sendPacketNow(session, packet);
+                } catch (Throwable e) {
+                    if (packet != null) {
+                        packet.release();
+                    }
+                    logger.info("response data now to session: {} failed", session, e);
```

Comment:
This seems to happen when an exception occurs, should we use `logger.error` instead?

Reply:
no, because there are a lot of cases make an exception happen, example a session disconnected from client, it's a normal case, so if we `error` log it can make spam logs on monitoring channel or tool (example slack)

Reply:
Yep, ok, sounds good to me